README.md

```diff
@@ -35,6 +35,10 @@ After that you can install it without problem.
 The package uses a prebuilt python virtual environnement. The binary are taken from this repository: https://github.com/Josue-T/synapse_python_build
 The script to build the binary is also available.
 
+### Performance Tuning
+
+Depending on the amount of RAM on your système, you might want to tune the cache configuration in `/etc/matrix-synapse/homeserver.yaml`.
```

Comment:
Maybe it would be good to put this in a specific file like `/etc/matrix-synapse/conf.d/cache.yaml` to avoid that this settings is always override by the update.

And the best solution is to give the possibility to edit this from the config pannel.